iocore/net/P_SSLNetVConnection.h

```diff
@@ -519,6 +532,8 @@ class SSLNetVConnection : public UnixNetVConnection, public ALPNSupport, public
 
   // Null-terminated string, or nullptr if there is no SNI server name.
   std::unique_ptr<char[]> _serverName;
+  std::unique_ptr<char[]> _ca_cert_file;
+  std::unique_ptr<char[]> _ca_cert_dir;
```

Comment:
These are needed because the SNI config can disappear during a transaction, destroying the original source, correct?